website/client/src/router/index.js

```diff
@@ -75,6 +75,8 @@ const GroupPlanIndex = () => import(/* webpackChunkName: "group-plans" */ '@/com
 const GroupPlanTaskInformation = () => import(/* webpackChunkName: "group-plans" */ '@/components/group-plans/taskInformation');
 const GroupPlanBilling = () => import(/* webpackChunkName: "group-plans" */ '@/components/group-plans/billing');
 
+const MessagesIndex = () => import(/* webpackChunkName: "group-plans" */ '@/pages/private-messages');
```

Comment:
why is this part of the group plans chunk? maybe it should have its own

Reply:
oops, fixed